Packs/QRadar/Integrations/QRadar_v3/QRadar_v3.py

```diff
@@ -35,7 +35,10 @@
 SAMPLE_SIZE = 2  # number of samples to store in integration context
 EVENTS_INTERVAL_SECS = 60  # interval between events polling
 EVENTS_MODIFIED_SECS = 5  # interval between events status polling in modified
+
 EVENTS_SEARCH_FAILURE_LIMIT = 3  # amount of consecutive failures events search will tolerate
+EVENTS_SEARCH_POLLING_RETRIES = 10  # number of consecutive failures events polling will tolerate
+EVENTS_SEARCH_RETRY_SECONDS = 100  # seconds between retries to poll offense events
```

Comment:
we should probably have parameters for 
EVENTS_SEARCH_POLLING_RETRIES,
EVENTS_SEARCH_RETRY_SECONDS

Reply:
Should we have parameters for it or use it with the advanced parameters?